results.php

```diff
@@ -7,29 +7,77 @@
 {
     $province = 'all';
 }
+$type = isset($_REQUEST['type']) ? $_REQUEST['type'] : 'all';
```

Comment:
Can we discuss a unified style for PHP? My default is to use the same kinds of style choices I would use when writing code in C, but if others have any arguments for some deviations to that style, let's discuss that here.

Reply:
So you don't do spaces in between () and []?

Reply:
No. I also prefer ternary expressions when they can be constrained to one or two lines. Additionally, if you look down below in the document, I prefer separating the if statement and the following brace with a space as opposed to a new line, and so on. Again, I'm willing to alter my style to match a different standard, but I think mine is fairly readable/acceptable.